### PR TITLE
BACKLOG-10919: Add createNewContentAction in CMM

### PR DESCRIPTION
**Pull request by FBerthelot**

### CODE REVIEW

File: src/javascript/actions/CreateNewContent/CreateNewContentDialog.jsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            label: 'D level1',
+            iconURL: 'https://image.flaticon.com/icons/svg/1973/1973617.svg'
+        }
+    ]
+};
+
+const styles = theme => ({
+    treeContainer: {
+        border: `1px solid ${theme.palette.ui.omega}`,
+        backgroundColor: '#F9FBFC',
```
Comment (dgriffon) [inline src/javascript/actions/CreateNewContent/CreateNewContentDialog.jsx]: Please use theme variables for colors and sizes

File: src/javascript/actions/CreateNewContent/CreateNewContentDialog.jsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+});
+
+const CreateNewContentDialogCmp = ({classes, t}) => {
+    const [isOpen, setIsOpen] = useState(true);
+    const closeModal = () => setIsOpen(false);
+
+    return (
+        <Dialog open={isOpen} aria-labelledby="dialog-createNewContent" onBackdropClick={closeModal}>
+            <DialogTitle id="dialog-createNewContent">
+                Create
```
Comment (dgriffon) [inline src/javascript/actions/CreateNewContent/CreateNewContentDialog.jsx]: Please use i18n framework for labels

File: src/javascript/register.jsx [hunk 3]
```diff
@@ -28,34 +35,48 @@ if (contextJsParameters && contextJsParameters.config && contextJsParameters.con
             mode: 'edit'
         });
 
+        actionsRegistry.add('createNewContent', createNewContentAction, {
+            buttonIcon: <Edit/>,
+            buttonLabel:
+                'content-editor:label.contentEditor.CMMActions.createNewContent.label',
+            target: ['createMenuActions:3.2', 'contentActions:3.2'],
+            showOnNodeTypes: ['jnt:contentFolder', 'jnt:content']
```
Comment (dgriffon) [inline src/javascript/register.jsx]: create content should be display only for `jnt:contentFolder` nodetype

File: src/javascript/actions/CreateNewContent/CreateNewContentDialog.jsx (hunk 1 above)
Comment (FBerthelot) [reply src/javascript/actions/CreateNewContent/CreateNewContentDialog.jsx]: I cannot see it in figma :/

You are right, I will ask directly to the designer !

**Pull request merged**